pkg/virt-handler/vm_test.go

```diff
@@ -2239,6 +2240,33 @@ var _ = Describe("VirtualMachineInstance", func() {
 			Expect(condition.Reason).To(Equal(v1.VirtualMachineInstanceReasonVirtIOFSNotMigratable))
 		})
 
+		It("should not be allowed to live-migrate if the VMI does use masquerade to connect to the pod network", func() {
```

Comment:
Missing a `not`: [...]live migration is forbidden if the VMI does `not` use [...]

Reply:
Fixed.